### PR TITLE
Lvjavs 405,432,568

### PR DESCRIPTION
**Pull request by VitalijF**
# **+**

### CODE REVIEW

File: src/main/webapp/resources/css/user-my-report.css [hunk 1]
```diff
@@ -143,4 +143,8 @@ min-width: 120px;
       background: #2196F3;
       color: white;
       border: 1px solid #2196F3;
-  }
```
Comment (merkan3k) [inline src/main/webapp/resources/css/user-my-report.css]: new line warning

File: src/main/webapp/resources/js/comment-modal-message.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Created by shuk on 16.11.16.
+ */
+function callCommentDialog()  {
+    $('[id^=comment]').click(function() {
+        var comment = $(this).attr('title');
+            $('#kidCommentMessage').find('h4').html(comment);
+            $('#kidCommentMessage').modal('show');
+    });
+}
```
Comment (merkan3k) [inline src/main/webapp/resources/js/comment-modal-message.js]: new line warning

File: src/main/webapp/resources/css/user-my-report.css (hunk 1 above)
Comment (VitalijF) [inline src/main/webapp/resources/css/user-my-report.css]: fixed

File: src/main/webapp/resources/js/comment-modal-message.js (hunk 2 above)
Comment (VitalijF) [inline src/main/webapp/resources/js/comment-modal-message.js]: fixed

**Comment by hrybs** [comment]
# **+**

File: src/main/java/ua/softserveinc/tc/constants/ValidationConstants.java [hunk 3]
```diff
@@ -76,6 +81,9 @@ private ValidationConstants() {
     public static final class ConfigFields {
         public static final String MIN_AGE = "kidsMinAge";
         public static final String MAX_AGE = "kidsMaxAge";
+        public static final String MAX_NAME_LENGTH = "kidsMaxNameLength";
+        public static final String MAX_COMMENT_LENGTH = "kidsMaxCommentLength";
+
```
Comment (Pitcity) [inline src/main/java/ua/softserveinc/tc/constants/ValidationConstants.java]: remove empty lines :P

File: src/main/java/ua/softserveinc/tc/dto/ConfigurationDto.java [hunk 4]
```diff
@@ -201,6 +223,11 @@ public boolean equals(Object o) {
         if (kidsMinAge != null ? !kidsMinAge.equals(that.kidsMinAge) : that.kidsMinAge != null) return false;
         if (kidsMaxAge != null ? !kidsMaxAge.equals(that.kidsMaxAge) : that.kidsMaxAge != null) return false;
 
+        if (kidsMaxNameLength != null ? !kidsMaxNameLength.equals(that.kidsMaxNameLength) :
+                that.kidsMaxNameLength != null) return false;
```
Comment (Pitcity) [inline src/main/java/ua/softserveinc/tc/dto/ConfigurationDto.java]: just wondering, why ConfigurationDto named as "that"?

**Pull request merged**